app-admin/mcelog/mcelog-194-r1.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="https://github.com/andikleen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
```

Comment:
You're losing stable keywords here? Either copy it to a new one and drop stable keywords, or git mv and keep the stable keywords.

Reply:
doing, soory. forgot about pkgbump

Reply:
`pkgcheck scan --commits` should flag this as `DroppedStableKeywords` too

Reply:
is it not recommended to rm the old ebuild when a new revision is made?